src/dev/src/app/vertical-nav/nested-menus/vertical-nav-nested-menus.ts

```diff
@@ -16,6 +16,7 @@ export class VerticalNavNestedMenusDemo {
 
   constructor(public verticalNavCases: VerticalNavCases) {
     this.case = this.verticalNavCases.allNestedMenu;
+    console.log(this.case);
```

Comment:
This is my log. There are many like it but this one is mine.

Reply:
Removed